<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>755</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>991</width>
    <height>755</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>991</width>
    <height>755</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QStackedWidget" name="main_stacked_widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>991</width>
      <height>571</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>991</width>
      <height>571</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>991</width>
      <height>571</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(216, 215, 188)</string>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="welcome_page">
     <property name="styleSheet">
      <string notr="true">QPushButton
{
font-family:'Harmonia Sans Pro, Lucia Bright,sans-serif';
font-size: 20px;
background-color:rgba(122,122,122,255);
color:white;
border-style:solid;
border-radius:10px;
}

QPushButton::Hover
{
color:rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(122, 122, 122, 255), stop: 0.501 rgba(122, 122, 122, 255), stop:1 rgba(255, 255, 255, 255));
border-color:rgb(255, 255, 255);
border-width:2px
}

QPushButton::Pressed
{
	border-color: rgb(0, 0, 0);
}</string>
     </property>
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>380</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Start Custom Trip</string>
      </property>
     </widget>
     <widget class="QLabel" name="welcome_label">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>70</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(31, 34, 31, .70);
border-radius: 10px;
color: white;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Welcome!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="splash_background">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-60</y>
        <width>1021</width>
        <height>691</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/new/prefix1/uclatest2.jpg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="transparent_box">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>340</y>
        <width>371</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(31, 34, 31, .70);
border-radius: 10px;
color: white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="transparent_box_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>340</y>
        <width>371</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(31, 34, 31, .70);
border-radius: 10px;
color: white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="ASU_trip_button">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>380</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>ASU Trip</string>
      </property>
     </widget>
     <widget class="QPushButton" name="shortest_trip_button">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>380</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Shortest Trip</string>
      </property>
     </widget>
     <zorder>splash_background</zorder>
     <zorder>welcome_label</zorder>
     <zorder>transparent_box</zorder>
     <zorder>startButton</zorder>
     <zorder>transparent_box_2</zorder>
     <zorder>ASU_trip_button</zorder>
     <zorder>shortest_trip_button</zorder>
    </widget>
    <widget class="QWidget" name="tours_page">
     <property name="styleSheet">
      <string notr="true">QPushButton
{
font: 75 12pt &quot;Sitka&quot;;
color: white;
background-color: rgba(31, 34, 31, .70);
border-style: solid;
border-color: black;
border-width: 5px;
border-radius:10px;
}

QPushButton::Hover
{
border-color: rgb(255, 255, 255);
}</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>721</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
     <widget class="QLabel" name="choosing_label">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>70</y>
        <width>511</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(31, 34, 31, .70);
border-radius: 10px;
color: white;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>10</number>
      </property>
      <property name="text">
       <string>Let's start with choosing our starting campus!</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="admin_login_page">
     <property name="styleSheet">
      <string notr="true">QPushButton
{
font: 75 12pt &quot;Sitka&quot;;
background-color: rgb(241, 241, 240);
border-style: solid;
border-color: black;
border-width: 2px;
border-radius:10px;
}

QPushButton::Hover
{
border-color: rgb(255, 255, 255);
}

QPushButton::Pressed
{
border-color: rgb(0, 0, 255);
}

QLabel
{
background-color: rgba(0, 61, 185, .01);
font: 75 12pt &quot;System&quot;;
}

QLineEdit
{
background-color: rgb(255, 255, 255);
}

</string>
     </property>
     <widget class="QLabel" name="transparent_box_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>170</y>
        <width>371</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 61, 185, .60);
border-radius: 10px;
color: white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="error_label">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>250</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400; color:#ff0000;&quot;&gt;*Error* - Username/Password is incorrect!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back_button">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>340</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="login_button">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>290</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_line_edit">
      <property name="geometry">
       <rect>
        <x>409</x>
        <y>218</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="username_line_edit">
      <property name="geometry">
       <rect>
        <x>409</x>
        <y>192</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="password_label">
      <property name="geometry">
       <rect>
        <x>332</x>
        <y>218</y>
        <width>68</width>
        <height>16</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="username_label">
      <property name="geometry">
       <rect>
        <x>332</x>
        <y>192</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLabel" name="admin_login_label">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>110</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="text">
       <string>Please enter your login credentials</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>150</y>
        <width>371</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>173</x>
        <y>30</y>
        <width>20</width>
        <height>521</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>30</y>
        <width>20</width>
        <height>521</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="maintenance_label">
      <property name="geometry">
       <rect>
        <x>236</x>
        <y>30</y>
        <width>501</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(31, 34, 31, .70);
border-radius: 10px;
color: white;
font: 75 bold 26pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Maintenance Login</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <zorder>transparent_box_3</zorder>
     <zorder>back_button</zorder>
     <zorder>login_button</zorder>
     <zorder>password_line_edit</zorder>
     <zorder>username_line_edit</zorder>
     <zorder>error_label</zorder>
     <zorder>username_label</zorder>
     <zorder>password_label</zorder>
     <zorder>admin_login_label</zorder>
     <zorder>line_3</zorder>
     <zorder>line_4</zorder>
     <zorder>line_5</zorder>
     <zorder>maintenance_label</zorder>
    </widget>
    <widget class="QWidget" name="maintenance_page">
     <property name="styleSheet">
      <string notr="true">QListWidget
{
background-color: rgba(31,31,31,.9);
border-style: solid;
border-color: black;
border-width: 5px;
border-radius: 10px;
color: white;
}
QPushButton
{
font: 75 12pt &quot;Sitka&quot;;
background-color: rgb(241, 241, 240);
border-style: solid;
border-color: black;
border-width: 2px;
border-radius:10px;
}

QPushButton::Hover
{
border-color: rgb(255, 255, 255);
}

QPushButton::Pressed
{
border-color: rgb(0, 0, 255);
}</string>
     </property>
     <widget class="QListWidget" name="souvenir_list_widget">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>70</y>
        <width>511</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>21</width>
        <height>541</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="exit_button">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>310</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>20</y>
        <width>21</width>
        <height>541</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>241</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
font: 75 12pt &quot;Sitka&quot;;
background-color: rgb(241, 241, 240);
border-style: solid;
border-color: black;
border-width: 2px;
border-radius:10px;
}

QPushButton::Hover
{
border-color: rgb(255, 255, 255);
}

QPushButton::Pressed
{
border-color: rgb(0, 0, 255);
}

QFrame
{
background-color: rgba(0, 61, 185, .60);
border-radius: 10px;
color: white;
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="add_souvenir_button">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>51</height>
          </size>
         </property>
         <property name="text">
          <string>Add Souvenir</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="upload_college_button">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>51</height>
          </size>
         </property>
         <property name="text">
          <string>Upload College from File</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="header_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>991</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>991</width>
      <height>141</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>991</width>
      <height>141</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Sitka Banner</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
background-color: rgb(84, 190, 255);
color: white;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;College Tour Planner&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="header_icon_push_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(84, 190, 255);
border-style:solid;
border-radius: 5px;
}

QPushButton::Hover
{
color:rgb(255, 255, 255);
border-color:rgb(255, 255, 255);
border-width:2px
}

QPushButton::Pressed
{
border-color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/new/prefix1/headerIcon.png</normaloff>:/new/prefix1/headerIcon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>130</width>
      <height>200</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="admin_push_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>1</y>
      <width>41</width>
      <height>44</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(84, 190, 255);
border-style:solid;
border-radius: 5px;
}

QPushButton::Hover
{
color:rgb(255, 255, 255);
border-color:rgb(255, 255, 255);
border-width:2px
}

QPushButton::Pressed
{
border-color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/new/prefix1/admin.png</normaloff>:/new/prefix1/admin.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="admin_label">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(84, 190, 255);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;ADMIN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="header_line">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>431</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(84, 190, 255);
</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
